packages/express/lib/log.js

```diff
@@ -32,7 +32,7 @@ morgan.token('pretty-response-time', (req, res) => {
   return prettyMS(Number(ms.toFixed()));
 });
 
-exports.loggerMiddleware = (logger) =>
+module.exports = (logger) =>
   morgan(':method :url :pretty-status :pretty-response-time - :pretty-size', {
     stream: split().on('data', (line) => logger.request(line)),
```

Comment:
I would like to not have `logger.request` but instead a more universal method like `logger.verbose` or something which could be used in other contexts too. Because `logger.request` suggests that it can only be used for requests and then people might think to use `logger.request` for their own request logging too.

Reply:
Imho the `info`-level already implies verbosity, so having a `logger.verbose`-method seems redundant to me. But `logger.request` might indeed be a little too specific and thus misleading. What about renaming it to something like `logger.plain`?

Reply:
How about a `logLevel` of `verbose` and a couple of different logger methods like `request` and `hint`? I want to keep the `[<project>:request]` prefix in our output.

Reply:
How about a `logger.custom(prefix, message): void` with a `logLevel` of `verbose`?

Reply:
Other loggers use `debug` or `silly` as method names for more verbose output.

Reply:
I think npm uses these levels:

```json
{ 
  error: 0, 
  warn: 1, 
  info: 2, 
  verbose: 3, 
  debug: 4, 
  silly: 5 
}
```

For `debug` output, we already have an existing mechanism I would like to keep around. And for religious reasons, I refuse to facilitate `silly` actions. ;)

Why don’t we go for these levels: 
```json
{ 
  error: 0, 
  warn: 1, 
  info: 2, 
  verbose: 3
}

Reply:
How about using a `Proxy` for generic log messages?

https://github.com/untool/untool/blob/20dfffdde33467f3fe0d357d31f75cf7a7dab77c/packages/yargs/lib/logger.js#L59-L64